gatsby-site/src/components/View/index.js

```diff
@@ -12,12 +12,9 @@ const View = (props) => {
       <Illustration />
       <ReactNavbar />
       <Jumbotron>
-        <div id="title">
           <h1 className="display-3">Building <br></br> better software, together.</h1>
```

Comment:
No need for br tags now that you fixed the word-breaks in body. Also, the word "focus" in the lead statement is mispelled.

Reply:
looks like the br tags need to stay for now...better will jump up to the same line as building and doesn't stay with software

Reply:
👍🏼